pkg/testutil/util.go

```diff
@@ -79,11 +79,18 @@ func IsYamlEqual(expectation *string, actualYamlByteArray *[]byte) bool {
 	}
 
 	// Replace Timestamps that would show up as different.
-	_, err := exec.Command("sed", "-r", "-i", ReplaceTimestampRegExp, actualYamlFileName).Output()
+	_, err := exec.Command("sed", "-r", "-i.tmp", ReplaceTimestampRegExp, actualYamlFileName).Output()
```

Comment:
we really shouldn't be shelling out in go code like this. Can you reimplement these `execs` with go libs?

Reply:
+1 for Brandon's comment in here... we need to implement these functions properly and thoroughly handle the returned errors.

Reply:
This is a temporary workaround in place which was discussed in a sagemaker-controller PR, we do plan to replace this in the future.